Kitodo/src/main/java/de/sub/goobi/export/dms/ExportDms.java

```diff
@@ -283,6 +287,7 @@ private boolean downloadImages(Process process, URI userHome, URI destinationDir
             if (exportDmsTask != null) {
                 exportDmsTask.setException(e);
             } else {
+                logger.error(e.getMessage(), e);
```

Comment:
There is already method (setErrorMessage), which takes inside logger and error. So in every place like that is better use setErrorMessage instead of adding additional line with logger.